ImperatorToCk2/Output.java

```diff
@@ -0,0 +1,737 @@
+                    out.println (tab+date1+"={");
+                    out.println (tab+tab+"create_bloodline = {");
+                    out.println (tab+tab+tab+"type = "+convTraitList.get(aqq).substring(0,convTraitList.get(aqq).length()-1));
+                    out.println (tab+tab+tab+"has_dlc = "+quote+"Holy Fury"+quote);
+                    out.println (tab+tab+"}");
+                    out.println (tab+"}");    
+                }else { //regular trait
+
+                    out.println (tab+"trait="+convTraitList.get(aqq));
+
+                }
+                aqq = aqq + 1;
+            }
+
+        }
+
+        if (sex != "69") { //if a character is dynamically generated or not
+            out.println (tab+"disallow_random_traits = yes");    
+        }
+
+        if (hasFather == 1) {
+            out.println (tab+"father="+father100);
+
+        }
+
+        if (hasMother == 1) {
+            out.println (tab+"mother="+mother100);
+
+        }
+
+        out.println (tab+birthday+"={");
+        out.println (tab+tab+"birth="+quote+birthday+quote);
+        out.println (tab+"}");
+
+        if (spouse != "0") {
+            out.println (tab+date2+"={");
+            out.println (tab+tab+"add_spouse="+spouse100);
+            out.println (tab+"}");
+        }
+
+        //default death date so the character will be dead in the 1066 start date
+        if (dead.equals("yes")) {
+            out.println (tab+date1+" ={");   
+        } else {
+            out.println (tab+"250.1.1 ={");    
+        }
+        out.println (tab+tab+"death= yes");
+        out.println (tab+"}");
+        out.println ("}");
+
+      
+        out.flush();
+        fileOut.close();
+
+        return irKING;
+    }
+
+    public static String dynastyCreation(String name, String id, String Directory) throws IOException
+    {
+
+        String VM = "\\"; 
+        VM = VM.substring(0);
+        char VMq = '"';
+        String tab = "	";
+
+        Directory = Directory + VM + "common" + VM + "dynasties";
+        String ck2CultureInfo ="a";   // Owner Culture Religeon PopTotal Buildings
+        FileOutputStream fileOut= new FileOutputStream(Directory + VM + "c_" + id + ".txt");
+        PrintWriter out = new PrintWriter(fileOut);
+
+        int flag = 0;
+
+        String date1 = "100.1.1";
+        String date2 = "1066.9.15";
+
+      
+        out.println (id+"=");
+        out.println ("{");
+        out.println (tab+"name="+VMq+name+VMq);
+        out.println (tab+"used_for_random=no");
+        out.println ("}");
+        out.flush();
+        fileOut.close();
+
+        return ck2CultureInfo;
+    }
+
+    public static String btitleCreation(String name, String Directory, int id) throws IOException
+    {
+
+        String VM = "\\"; 
+        VM = VM.substring(0);
+        char VMq = '"';
+        String tab = "	";
+        String oldName = name;
+        name = name.toLowerCase();
```

Comment:
name -> cleanName or something of the sort (make sure to update the place where it's used).